packages/core/test/userapp/index.js

```diff
@@ -26,6 +26,24 @@ const List = {
   },
 };
 
+const ListRequire = {
+  key: 'listrequire',
+  noun: 'List',
+  list: {
+    display: {
+      description: 'Trigger on new thing in list.',
+      label: 'List',
+    },
+    operation: {
+      perform: (z, bundle) => {
+        // in prod, process.cwd will return the app root directory
+        const { BASE_URL } = z.require('./test/userapp/constants.js');
```

Comment:
@eliangcs @pragmatic-zac  👋 I'd appreciate your thoughts here - would this test be sufficient? I'd prefer to have it referencing `./constants.js` directly since that's how it would be in an actual app/app extension, but I tried various ways of mocking `process.cwd` (mostly using `mock-fs`) and ran into issues getting it to recognize the module 🤔 If this test should be made more robust, would `mock-fs` also be y'all's go-to, or is there another way I should explore of mocking that when `z.require()` is used?

Reply:
Yes, I think this is sufficient. Thanks for adding the test! Approving.

I was wondering if there's a better way to locate the app root directory other than `process.cwd`, but I couldn't find one, so 🤷 .